### PR TITLE
Feat added new debt form

### PR DESCRIPTION
**Pull request by Pr0bka**

### CODE REVIEW

File: src/components/DebtForm/DebtForm.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  const debtData = {
+    sender: sender,
+    receiver: receiver,
+    currency: enteredCurrency,
+    sum: enteredSum + ` ${enteredCurrency}`,
+    description: enteredDescription,
+    date: EnteredDate,
+  };
+
+  const clearForm = () => {
```
Comment (AntonOvc) [inline src/components/DebtForm/DebtForm.js:34]: cannot clean up sender and receiver? 😄 please add functionality either here  or in subsequent PR

File: src/routes/AddTransactionPage.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
 
 const AddTransactionPage = (props) => {
   return (
     <Fragment>
       <label className={classes.addDebtlabel}>Кто кому дает за щеку?</label>
-      <div>
-        <AvatarsList/>
-        <AvatarsList/>
-      </div>
+        <DebtForm></DebtForm>
```
Comment (AntonOvc) [inline src/routes/AddTransactionPage.js:10]:
```suggestion
        <DebtForm/>
```

**Review by AntonOvc — CHANGES_REQUESTED** [review]
Looks good! please check comments and merge

File: src/components/DebtForm/DebtForm.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
+import { Fragment, useState } from 'react';
+import classes from './DebtForm.module.css';
+import AvatarsList from '../AvatarsList/AvatarsList';
+
+const DebtForm = () => {
+  const [sender, setSender] = useState();
+  const [receiver, setReceiver] = useState();
+  const [enteredCurrency, setEnteredCurrency] = useState(``);
+  const [enteredSum, setenteredSum] = useState(``);
```
[the remark this answers is not in the document]
Comment (Pr0bka) [reply src/components/DebtForm/DebtForm.js:9]: Fixed

File: src/components/DebtForm/DebtForm.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  const [enteredDescription, setEnteredDescription] = useState(``);
+  const [EnteredDate, setEnteredDate] = useState(``);
```
[the remark this answers is not in the document]
Comment (Pr0bka) [reply src/components/DebtForm/DebtForm.js:11]: Fixed

File: src/routes/AddTransactionPage.js (hunk 2 above)
Comment (Pr0bka) [reply src/routes/AddTransactionPage.js:10]: fixed

File: src/components/DebtForm/DebtForm.js (hunk 1 above)
Comment (Pr0bka) [reply src/components/DebtForm/DebtForm.js:34]: later

**Review by AntonOvc — APPROVED** [review]

**Pull request merged**